lint-staged.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  '*.css': ['prettier --write', 'stylelint --fix'],
+  '*.json': 'prettier --write',
+  '*.md': 'prettier --write',
+  '*.mdx': 'prettier --write',
+  '*.ts?(x)': () => 'yarn lint-ts',
+  '*.{js,jsx,ts,tsx}': ['prettier --write', 'eslint --fix']
+}
```

Comment:
New `lint-staged` config that handles every file type with at least `prettier`, while also running `stylelint` and `eslint` when applicable. All commands are fitted with the options to automatically fix issues instead of reporting and erroring.